KNOWN_ISSUES.md

```diff
@@ -60,11 +60,11 @@ If usage values are equal zero, means that there is problem is related to Kubele
 
 **Known causes**
 
-* Incorrect configuration Kubelet integration with container runtime 
-* Container runtime doesn't support [container metrics RPCs](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-node/cri-container-stats.md) or doesn't have [cAdvisor](https://github.com/google/cadvisor) support.
```

Comment:
Does this `cause` need to be removed? Or where could we record it?  
/retest

Reply:
Those are theoretical causes that I added to give some hints to reader. With this PR I wanted to replace theroretical ones with concrete one based on https://kubernetes.slack.com/archives/C20HH14P7/p1643137790031400?thread_ts=1642595289.013400&cid=C20HH14P7

I think using concrete causes is better as it gives clear action items to users.

Reply:
/lgtm